### ISSUE TITLE
new page has incorrect extension, fails tests.

### ISSUE THREAD

**Issue by coke** [Raku/doc#4007]
b926ebab6 added doc/Language/filename-extensions.rakudoc, which isn't going to be indexed or work with any other util/ tooling. Needs to be changed to .pod6

Additionally, this new file fails "make xtest".

**Comment by JJ** [comment]
The thing is, it's not really incorrect. I really wondered, when it was accepted, if something was going to fail. Now the question is, should we change the extension or change the tooling?

**Comment by coke** [comment]
There’s another ticket (I think) for renaming all the pod6 tickets. No need to support both simultaneously. But keep them consistent until the cutover and do them all at once.

**Comment by JJ** [comment]
Why not? They're both valid extensions for the time being...

**Comment by coke** [comment]
Because this is the simplest solution to get the page tested and part of the site. I am trying not to block adding this content on retooling everything to allow multiple extensions.

**Comment by coke** [comment]
#3785

**Comment by coke** [comment]
(test failures) note that the test files only consider pod6 at the moment, so with the current name, many test files are skipped; When the file is renamed, several tests that were skipped then fail.

**Comment by JJ** [comment]
Let's try to fix the tests for the time being, and then decide on change, would that be OK for you?

**Comment by coke** [comment]
If it's something other than "rename the file", no, it's not OK for me.

**Comment by coke** [comment]
I don't understand if you mean "rename the file and make the tests pass" (which is what I proposed, so yes), or if you mean "leave the file name as is and make the the tests understand the new name", which as I've tried to explain, is covered by another ticket, requires more work, and was argued against when it was proposed.

**Comment by JJ** [comment]
I meant rename the file to make it be checked by the tests, then fix whatever is wrong, and then rename it back. But really, it makes no sense. So I guess it's renaming it and then fix whatever is wrong. I'll do that.

**Comment by finanalyst** [comment]
I'll rename file.
I was just trying to be consistent with the content of the file.

**Comment by coke** [comment]
I took the next step of making the new file pass tests. You can do this pre-commit by running:

```
TEST_FILES="path/to/file1.pod6 path/to/file2.pod6" make xtest
```

**Comment by JJ** [comment]
Please close when it's actually fixed... I can't reopen, for some reason.

**Comment by finanalyst** [comment]
I renamed file.

**Comment by JJ** [comment]
So close the issue from that commit if that's the case; as indicated by @coke there were other issues besides the name, also.

**Comment by finanalyst** [comment]
I'm confused.

This issue was about renaming a file (ie. file-extensions.rakudoc -> <...>.pod6). I renamed it and merged the commit. The file is now renamed in the docs/ directory. 

I closed this issue after merging the file renaming.

@coke has subsequently modified the file to change spellings. He also added .crotmp to another test file config. I think he re-opened this issue to provide feedback. ????

Confusingly: In error, my first attempt at a file rename also polluted another PR I have waiting Tom Browder's review. I reverted that rename, and then created the rename commit via a temporary branch, which I merged immediately.

So which issue / commit needs closing?
